addons/point_of_sale/static/src/app/models/pos_order_line.js

```diff
@@ -618,6 +618,13 @@ export class PosOrderline extends Base {
                 this.getUnitDisplayPriceBeforeDiscount(),
                 this.currency
             ),
+            taxGroupLabels: [
+                ...new Set(
+                    this.product_id.taxes_id
+                        ?.map((tax) => tax._pos_receipt_label)
+                        .filter((label) => label)
```

Comment:
In case of a partial implementation of the label (like without 2 taxes one with label and not the other), I think you should fallback on the tax name... or do you have something somewhere preventing this?

Reply:
I hope you challenged the most complex taxes combinations we know so far and validated them functionally like the 2 x 2.5% + 1% in india or the 5 x division taxes in brazil. Do you merge them according the tax groups per line?

Reply:
For the first point, we can't fallback to the tax name as it can be long which will mess up the formatting. The main idea is to setup the **tax group labels** using the **.csv** files so they are pre-filled automatically. but just in case the regulations changed, we give the user the option to modify it in the **debug** mode. So we won't enforce the existence of **tax group label** on all countries. however it can be individually enforced per country if needed by regulations

For the second point, we just care about assigning a label to to the **tax group** of the **tax**. For complex configs. It has to be individually formatted like INDIA. the PO requested to keep the current breakdown alongside the new -which this PR introduces- alongside!

The purpose of this PR is to introduce a generalised approach.